arch/arm/src/nrf53/Kconfig

```diff
@@ -188,6 +199,64 @@ endchoice
 
 endmenu # System Timer
 
+menu "SAADC Configuration"
+
+if NRF53_SAADC
+
+choice
+	prompt "SAADC trigger selection"
+	default NRF53_SAADC_TASK
+	---help---
+		Choose mode for sample rate control
+
+config NRF53_SAADC_TASK
+	bool "SAADC Task trigger"
+
+config NRF53_SAADC_TIMER
+	bool "SAADC Timer trigger"
+
+endchoice # SAADC trigger selection
+
+if NRF53_SAADC_TIMER
+
+config NRF53_SAADC_TIMER_CC
+	int "SAADC Timer CC"
+	default 0
+	range 80 2047
+
+endif #NRF53_SAADC_TIMER
+
+config NRF53_SAADC_OVERSAMPLE
+	int "SAADC oversample"
+	default 0
+	range 0 8
+	---help---
+		SAADC oversample control
+
+config NRF53_SAADC_RESOLUTION
+	int "SAADC resolution"
+	default 0
+	range 0 3
+	---help---
+		SAADC resolution 0 - 8 bits, 1 - 10 bits, 2 - 12 bits, 3 - 14 bits
```

Comment:
I think the menu choice would be more user friendly, but we can go with help explanation as initial approach